test/e2e/user-actions-benchmark.js

```diff
@@ -0,0 +1,143 @@
+const path = require('path');
+const { promises: fs } = require('fs');
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
+const { exitWithError } = require('../../development/lib/exit-with-error');
+const {
+  isWritable,
+  getFirstParentDirectoryThatExists,
+} = require('../helpers/file');
+const { convertToHexValue, withFixtures } = require('./helpers');
```

Comment:
Imports can be organised better.

Reply:
Ty for the suggestion!! I've sorted them @jpuri